jdaviz/configs/default/plugins/collapse/tests/test_collapse.py

```diff
@@ -18,39 +18,12 @@ def test_linking_after_collapse(spectral_cube_wcs):
 
     coll.selected_data_item = 'test'
 
-    coll.selected_axis = 0
     coll.add_replace_results = False
     coll.vue_collapse()
 
     assert len(dc) == 2
     assert dc[1].label == 'Collapsed 1 test'
-    assert len(dc.external_links) == 3
+    assert len(dc.external_links) == 2
```

Comment:
The results changed here but I don't know what it means. Plugin seems to work though. 🤷